<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>728</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>fsensor v1.0</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushButton_run">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>50</y>
      <width>101</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>수집</string>
    </property>
   </widget>
   <widget class="QLabel" name="label1">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>180</y>
      <width>320</width>
      <height>240</height>
     </rect>
    </property>
    <property name="text">
     <string>lable1l</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_id">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>50</y>
      <width>141</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_state">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>80</y>
      <width>141</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>430</y>
      <width>320</width>
      <height>240</height>
     </rect>
    </property>
    <property name="text">
     <string>lable2</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>50</y>
      <width>56</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>이름:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>80</y>
      <width>56</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>상태:</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressBar1">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>130</y>
      <width>321</width>
      <height>16</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="value">
     <number>0</number>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>0</y>
      <width>341</width>
      <height>41</height>
     </rect>
    </property>
    <property name="title">
     <string>Mode</string>
    </property>
    <widget class="QRadioButton" name="radioButton_local">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>10</y>
       <width>108</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string>Local</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_remote">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>10</y>
       <width>108</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string>Remote</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>400</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
